src/languages/es.ts

```diff
@@ -676,6 +676,7 @@ export default {
         importFailedTitle: 'Fallo en la importación',
         importFailedDescription: 'Por favor, asegúrate de que todos los campos estén llenos correctamente e inténtalo de nuevo. Si el problema persiste, por favor contacta a Concierge.',
         importCategoriesSuccessfullDescription: (categories: number) => (categories > 1 ? `Se han agregado ${categories} categorías.` : 'Se ha agregado 1 categoría.'),
+        importMembersSuccessfullDescription: (members: number) => (members > 1 ? `Se han agregado ${members} miembros.` : 'Se ha agregado 1 miembro.'),
```

Comment:
These translations are already approved?

Reply:
I'm not quite sure and I can't find a message requesting a translation approval, I'll check this!

Reply:
I believe that Filip checked the translations before previous PR that handled `Categories` import, these are generally the same except 'Members' and 'Tags' names